pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,42 @@
+{ 
+  pkgs ? import <nixpkgs> {}
+  , lib ? pkgs.lib
+  , stdenv ? pkgs.stdenv
+  , makeWrapper ? pkgs.makeWrapper
+  , fetchFromGitHub ? pkgs.fetchFromGitHub
+  , fakeSha256 ? pkgs.lib.fakeSha256
+  , cmake ? pkgs.cmake
+  , pkg-config ? pkgs.pkg-config
+  , cryptopp ? pkgs.cryptopp
+  , qtbase ? pkgs.qt5.qtbase
+}:
+
+stdenv.mkDerivation {
+
+  name = "OTPGen";
+
+  meta = {
+    description = "MULTI-PURPOSE OTP TOKEN GENERATOR WRITTEN IN C++ AND QT WITH ENCRYPTION AND SYSTEM TRAY ICON. SUPPORTS TOTP, HOTP, AUTHY AND STEAM AND CUSTOM DIGIT LENGTHS AND PERIODS.";
+    license = lib.licenses.mit;
+    homepage = "https://github.com/magiruuvelvet/OTPGen";
+    maintainers = [ "Scott Hamilton <sgn'dot'hamilton'at'protonmail'dot'com>" ];
+    priority = "10";
```

Comment:
Please do not set a priority unless you specifically need to resolve conflicts.

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/5664fd4112b3dfedc09e489d46a269f97d566a61